tests/cucumber/steps/steps.js

```diff
@@ -2157,9 +2157,20 @@ module.exports = function getSteps(options) {
   );
 
   When(
-    'we make a GetApplicationBoxes call for applicationID {int} with max {int}',
-    async function (index, limit) {
-      await this.v2Client.getApplicationBoxes(index).max(limit).doRaw();
+    'we make a GetApplicationBoxes call for applicationID {int} with max {int} prefix {string} next {string} values {string}',
+    async function (index, limit, prefix, next, valuesAsString) {
+      let values = false;
+      if (valuesAsString === 'true') {
+        values = true;
+      }
```

Comment:
```suggestion
      const values = (valuesAsString === 'true')
```

Reply:
+1 here and below

Reply:
Thanks, addressed both places in latest push.